src/redux/configureStore.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
- This is an example of how the to configure store could use redux dev tools, this could help to know how the requests are made it on the back-end, I invite you to add this as another tool to use when working on React.js.
<img width="772" alt="image" src="https://user-images.githubusercontent.com/20563252/192666571-edbc3be5-a8f7-444e-844b-4dffdeaf53ac.png">

<img width="1693" alt="image" src="https://user-images.githubusercontent.com/20563252/192666604-795a8ba5-8b1a-45cb-be66-60e359a6dc03.png">

Reply:
- Check this [Chrome extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=es) to know more about this.

Reply:
Thanks for your comments! We are working with logger and strict mode.